EssentialFeed/EssentialFeediOS/Comments UI/Views/FeedCommentCell.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  FeedCommentCell.swift
+//  EssentialFeediOS
+//
+//  Created by Danil Vassyakin on 3/31/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import UIKit
+
+public final class FeedCommentCell: UITableViewCell {
+	
+	@IBOutlet weak var authorNameLabel: UILabel!
+	@IBOutlet weak var commentTimeLabel: UILabel!
+	@IBOutlet weak var commentTextLabel: UILabel!
```

Comment:
The setters can be private to prevent clients from mutating the properties:

```suggestion
	@IBOutlet private(set) weak var authorNameLabel: UILabel!
	@IBOutlet private(set) weak var commentTimeLabel: UILabel!
	@IBOutlet private(set) weak var commentTextLabel: UILabel!
```